src/main.js

```diff
@@ -32,6 +33,19 @@ const run = () => {
     fs.writeFileSync('ta.key', tlsAuthKey)
   }
 
+  // client dsn Script Path
+  if (dnsHelperScriptFile.length > 0) {
+    if (!fs.existsSync(dnsHelperScriptFile)) {
+      throw new Error(`dns helper script file '${dnsHelperScriptFile}' not found`)
+    }
+    const githubWorkspacePath = process.env.GITHUB_WORKSPACE
+
+    fs.appendFileSync(configFile, 'script-security 2\n')
+    fs.appendFileSync(configFile, `up ${githubWorkspacePath}/${dnsHelperScriptFile}\n`)
```

Comment:
It is recommended to call `up-restart` here.
cf. https://github.com/jonathanio/update-systemd-resolved#openvpn-configuration